app/assets/javascripts/listings/ListingService.js.coffee

```diff
@@ -606,7 +606,8 @@ ListingService = ($http, $localStorage, $q, $state, $translate, ModalService, Ex
     Service.stubListingPreferences()
     # if this listing had stubbed preferences then we can abort
```

Comment:
This comment is incorrect. we abort if the listing preferences are already defined. 

Is this OK to do? Is there a case where the listing preferences could be defined, but for the wrong listing? Should we instead set them to null at the start of this function, then always call?

Reply:
I verified that if you navigate to a different listing url, that the preferences refresh as expected, but I'm not sure if I'm missing anything